weblogic/tox.ini

```
@@ -0,0 +1,28 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py27-{12,14}
+    py38-{12,14}
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py38: {toxworkdir}/py38
+description =
+    py38: e2e ready
```

```suggestion
envdir =
    py27: {toxworkdir}/py27
    py38: {toxworkdir}/py38
description =
   py{27,38}: e2e ready
```
Should it include py27?

---

Yup, I've added it in now